python/mujinplanningclient/binpickingplanningclient.py

```diff
@@ -46,641 +75,2942 @@ def __init__(self, regionname=None, **kwargs):
+            taskparameters['debuglevel'] = debuglevel
         taskparameters.update(kwargs)
         return self.ExecuteCommand(taskparameters, robotspeed=robotspeed, toolname=toolname, timeout=timeout)
 
-    def PausePickPlace(self, timeout=10, **kwargs):
+    def PausePickPlace(self, timeout=10, dynamicEnvironmentState=None, debuglevel=None, **kwargs):
+        # type: (float, Optional[types.PausePickPlaceParametersDynamicEnvironmentState], Optional[int], Optional[Any]) -> Optional[Any]
         """
         Args:
-            timeout (float, optional): Time in seconds after which the command is assumed to have failed. (Default: 10)
+            timeout: Time in seconds after which the command is assumed to have failed. (Default: 10)
+            dynamicEnvironmentState: Dynamic environment state that allows the user to set/create objects in a particular state dynamically. (Default: None)
+            debuglevel: Sets the debug level for the planning logs. For development. 3=INFO, 4=DEBUG, 5=VERBOSE. (Default: None)
         """
         taskparameters = {
             'command': 'PausePickPlace',
-        }
+        }  # type: dict[str, Any]
+        if dynamicEnvironmentState is not None:
+            taskparameters['dynamicEnvironmentState'] = dynamicEnvironmentState
+        if debuglevel is not None:
+            taskparameters['debuglevel'] = debuglevel
         taskparameters.update(kwargs)
         return self.ExecuteCommand(taskparameters, timeout=timeout)
 
-    def ResumePickPlace(self, timeout=10, **kwargs):
+    def ResumePickPlace(self, timeout=10, dynamicEnvironmentState=None, debuglevel=None, numTargetsToPlace=None, enablerobotbridge=None, **kwargs):
+        # type: (float, Optional[types.ResumePickPlaceParametersDynamicEnvironmentState], Optional[int], Optional[int], Optional[bool], Optional[Any]) -> Optional[Any]
         """
         Args:
-            timeout (float, optional): Time in seconds after which the command is assumed to have failed. (Default: 10)
+            timeout: Time in seconds after which the command is assumed to have failed. (Default: 10)
+            dynamicEnvironmentState: Dynamic environment state that allows the user to set/create objects in a particular state dynamically. (Default: None)
+            debuglevel: Sets the debug level for the planning logs. For development. 3=INFO, 4=DEBUG, 5=VERBOSE. (Default: None)
+            numTargetsToPlace: The number of targets that are left to place (Default: None)
+            enablerobotbridge: If True, enables robot bridge (Default: None)
         """
         taskparameters = {
             'command': 'ResumePickPlace',
-        }
+        }  # type: dict[str, Any]
+        if dynamicEnvironmentState is not None:
+            taskparameters['dynamicEnvironmentState'] = dynamicEnvironmentState
+        if debuglevel is not None:
+            taskparameters['debuglevel'] = debuglevel
+        if numTargetsToPlace is not None:
+     
```

Comment:
fireandforget disappeared

Reply:
Because it wasn't ever defined, this would have errored out if it had been used :roll_eyes: